packages/picker/README.md

```diff
@@ -431,20 +516,32 @@ When the `value` of an `<sp-picker>` matches the `value` attribute or the trimme
 </sp-picker>
 ```
 
-### Pending
+</sp-tab-panel>
+</sp-tabs>
+
+#### Pending
 
 When in pending state, `<sp-picker>` elements will not respond to click events and will be delivered with a `<sp-progress-circle>` to visually outline that it is pending. It will not toggle open or display its `<sp-menu-item>` descendants until the attribute is removed.
```

Comment:
```suggestion
While in pending state, `<sp-picker>` elements will not respond to click events and will be delivered with an `<sp-progress-circle>` to visually denote that it is pending. It will not toggle open or display its `<sp-menu-item>` descendants until the attribute is removed.
```